samples/hosting/docker/sample.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Hosting endpoints in Docker containers
```

Comment:
Since there are both linux and windows containers, should we have some explanation about which of these this sample will work with? 

While the `microsoft/dotnet:runtime` is [multi-arch](https://github.com/dotnet/announcements/issues/14), the rabbitmq image isn't, so I believe that means this sample will only work with linux containers.

Reply:
I will make sure this is explained properly. Great comment!